connect/transforms/src/main/java/org/apache/kafka/connect/transforms/SetSchemaMetadata.java

```diff
@@ -39,18 +39,22 @@ public abstract class SetSchemaMetadata<R extends ConnectRecord<R>> implements T
     public static final String OVERVIEW_DOC =
             "Set the schema name, version or both on the record's key (<code>" + Key.class.getName() + "</code>)"
                     + " or value (<code>" + Value.class.getName() + "</code>) schema.";
-
```

Comment:
Let's keep this empty line that separates the fields from the interface but remove the new empty line added just below.